specification/postgresql/Microsoft.OrionDB.Management/examples/2025-08-01-preview/Clusters_CreateOrUpdate.json

```diff
@@ -0,0 +1,203 @@
+{
+  "operationId": "Clusters_CreateOrUpdate",
+  "title": "Create or update an OrionDB cluster",
+  "parameters": {
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "resourceGroupName": "TestGroup",
+    "orionDbClusterName": "testcluster1",
+    "api-version": "2025-08-01-preview",
+    "parameters": {
+      "location": "eastus",
+      "tags": {
+        "Environment": "Development",
+        "Team": "Database"
+      },
+      "properties": {
+        "createMode": "Default",
+        "administratorLogin": "testadmin",
+        "administratorLoginPassword": "password123!",
+        "version": "16",
+        "availabilityZone": "1",
+        "storage": {
+          "storageSizeGB": 512,
+          "autoGrow": "Enabled",
+          "tier": "P10"
+        },
+        "authConfig": {
+          "activeDirectoryAuth": "Enabled",
+          "passwordAuth": "Enabled",
+          "tenantId": "ffffffff-ffff-ffff-ffff-ffffffffffff"
+        },
+        "dataEncryption": {
+          "type": "SystemManaged"
+        },
+        "backup": {
+          "retentionDays": 7,
+          "geoRedundantBackup": "Disabled"
+        },
+        "network": {
+          "publicNetworkAccess": "Enabled",
+          "delegatedSubnetResourceId": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/TestGroup/providers/Microsoft.Network/virtualNetworks/test-vnet/subnets/default",
+          "privateDnsZoneResourceId": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/TestGroup/providers/Microsoft.Network/privateDnsZones/testcluster1.private.postgres.database.azure.com"
+        },
+        "highAvailability": {
+          "mode": "ZoneRedundant",
+          "standbyAvailabilityZone": "2"
+        },
+        "maintenanceWindow": {
+          "customWindow": "Enabled",
+          "dayOfWeek": 0,
+          "startHour": 2,
+          "startMinute": 0
+        }
+      },
+      "sku": {
+        "name": "Standard_D4s_v3",
+        "tier": "GeneralPurpose"
```

Comment:
We aren't exposing SKU and tier right - need to sync with Yogesh this is true of everywhere there is SKU